framework/org.eclipse.concierge/test/org/eclipse/concierge/BundleGetDataFileTest.java

```diff
@@ -55,6 +55,17 @@ private void setupDefaultBundles() throws Exception {
 		builder.addManifestHeader("Fragment-Host", "bundle");
 		fragmentUnderTest = installBundle(builder);
 	}
+	private static String joinString(String delimiter, String... args){
```

Comment:
Maybe naming this method `joinPath` and use as delimiter the File.separator within the method? This would simplify the use of that method.
```
private static String joinPath(String... args) {
    // ...
    sb.append(File.separator);
    // ...
```

Reply:
I agree with your naming. This method should be `joinPath`.